backend/src/litigation_api/ingestion/classification.py

```
@@ -0,0 +1,73 @@
+
+class ClassificationProvider(abc.ABC):
+    @abc.abstractmethod
+    def classify(self, text: str, title: Optional[str] = None, docket_title: Optional[str] = None) -> Optional[str]:
+        pass
+
+class DocumentClassifier:
+    """
+    Classifies documents into the supported v1 filing types using deterministic rules
+    with explicit fallback behavior.
+    """
+    def __init__(self, fallback_provider: Optional[ClassificationProvider] = None):
```

Ruff (0.15.11)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)